app/Http/Controllers/Product/ProductController.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace App\Http\Controllers\Product;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\Product;
+use App\Brand;
+use App\Category;
+
+class ProductController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $products = Product::all()->where('seller_id',auth()->id());
+        return view('seller.product.index_product')->with('products',$products);
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        $brands = Brand::get_available_brands();
+        $categories = Category::all();
+        return view('seller.product.create_product')->with('brands',$brands)->with('categories',$categories);
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $request->validate(
+            [
+            'title'=>'required|unique:products|min:3|max:25',
+            'description'=>'required|max:110',
+            'in_stock_quantity' =>'required|max:500',
+            'image' => 'required|image|mimes:jpeg,png,jpg',
+            'price'=>'required'
+            ]
+        );
+        $product = Product::create([
+            'title' => $request->input('title'),
+            'description' => $request->input('description'),
+            'in_stock_quantity' => $request->input('in_stock_quantity'),
+            'brand_id'=>$request->input('brand_id'),
+            'category_id'=>$request->input('category_id'),
+            'seller_id' =>auth()->id(),
+            'price' => $request->input('price'),
+        ]);
+       
+        $image = $request->file('image');
+        $name = $product->id.'.'.$image->extension();
+        $destination = public_path('/productImages');
+        $image->move($destination,$name);
+        $product->image = $name;
+        $product->save();
+       
+    
+            return redirect()->action('Product\ProductController@index');
```

Comment:
After creation of a project (if saving was successful) the user should be redirected to the single product page and not to the index (you'll need to use the show method and not the index method).